src/index.ts

```diff
@@ -178,13 +174,17 @@ async function appBoot() {
 	}
 
 	app.on( 'ready', async () => {
+		// Set translations based on supported locale
+		const locale = getSupportedLocale();
+		const localeData = getLocaleData( locale );
+		defaultI18n.setLocaleData( localeData?.locale_data?.messages );
```

Comment:
The supported locale uses `app.getLocale` that is only available when the app is read.